AdblockPlusSafari/AdblockPlusExtras.m

```diff
@@ -183,28 +184,54 @@ - (void)checkActivatedFlag
   }
 }
 
-- (void)updateFilterLists:(BOOL)userTriggered
+- (void)updateAllFilterLists:(BOOL)userTriggered
 {
-  NSMutableDictionary *filterLists = [self.filterLists mutableCopy];
-  for (NSString *filterListName in self.filterLists) {
+  [self updateFilterLists:self.filterLists userTriggered:YES];
+}
+
+- (void)updateFilterLists:(NSDictionary<NSString *, NSDictionary<NSString *, id> *> *)filterLists userTriggered:(BOOL)userTriggered
+{
+  NSMutableDictionary *mutableFilterLists = [self.filterLists mutableCopy];
```

Comment:
self.filterLists is defined as NSDictionary so it is not mutable. But it contain mutable instances of dictionaries, but I do not use them like that. self.filterLists is meant to be immutable.